app/reducers/index.js

```diff
@@ -111,6 +111,37 @@ export const addRatingToCurrentGoal = ({ currentGoal, }, newRating) => {
 export const selectRatingById = (id, arr) =>
   arr.find(rating => rating.id === id) || null;
 
+export const removeGoalFromArray = (state, { goal, }, fn = goal => goal) => {
```

Comment:
since we're no longer removing the goal from the array, might be best to rename this one?